src/providers/PeopleProvider.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  FC,
+  PropsWithChildren,
+  createContext,
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { useParams } from 'react-router-dom';
+import { usePeopleParams } from '../hooks/usePeopleParams';
+import { getCentaury } from '../utils/peopleUtils';
+
+interface PeopleContextI {
```

Comment:
if you want to use `I` inside your interface name we typically use it as a prefix 😉 
```suggestion
interface IPeopleContext {
```